.gitignore

```diff
@@ -26,3 +26,6 @@ go.work
 
 # GolangCI-Lint cache
 /cache
+
+# goland
+.idea
```

Comment:
I'd suggest to remove these IDE specific files from gitignore. More at https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
TIL, thanks!